lib/src/setupTests.ts

```diff
@@ -8,6 +8,12 @@
 
 import 'jest-enzyme';
 
+// We can't have @testing-library/jest-dom imported globally
+// because it will overwrite some of the jest-enzyme matchers.
+// Instead, we need to import them individual in the test modules
+// until we fully transitioned to react-testing-library.
+// import '@testing-library/jest-dom';
```

Comment:
We also had conflicts issue with `jasmineEnzyme` matchers in IQ.
The way we resolved it is:
- we load `jasmineEnzyme` matchers in `beforeAll`:
   https://github.com/sonatype/insight-brain/blob/71b8750d19838df221e9a76e004bf228aaf0fe51/insight-brain-frontend/src/test/frontend/specRoot.js#L62

- and then for RTL tests we override the matchers in `render` wrapper:
  https://github.com/sonatype/insight-brain/blob/722ced204aeb16ebd4a950e52e63668ed2e8be22/insight-brain-frontend/src/test/frontend/SpecUtil.js#L349

`render` wrapper is recommended practice if you need to perform some additional set-up that should apply to all your RTL tests. We use it to wrap components under test with Redux store provider.